web_responsive/static/src/less/navbar.less

```diff
@@ -18,12 +18,159 @@
         padding: @app-drawer-navbar-padding-vertical @app-drawer-padding-horizontal;
     }
     
+    // App name before oe_submenu
+    .app-name {
+        > .oe_menu_text {
+            font-size: 20px;
+            position: relative;
+            padding: 0 8px;
+            top: 1px;
+        }
+    }
+
+    // Fitt's law: Stick menu to the top of the screen
+    @media (min-width: @screen-sm) {
+        .navbar-nav {
+            margin-top: -2px;
+        }
+    }
+
+    .navbar-header {
+        // Navbar header mobile styling
+        margin-left: -11px;
+        margin-top: 5px;
+        margin-bottom: 5px;
+        margin-right: -11px;
+        .navbar-toggle {
+             margin: 0;
+        }
+
+        // Fitt's law: Stick app-drawer button to the top-left of the screen
+        @media (min-width: @screen-sm) {
+            margin-left: -16px;
+            margin-right: 0px;
+            margin-bottom: 0px;
+            margin-top: 0px;
+            padding: 0px;
+            padding-right: 0px;
+        }
+    }
+
+    @media (max-width: @screen-xs-max) {
+        ul.nav.navbar-nav.navbar-right.navbar-systray-item.oe_systray {
+            margin-top: 1.5px;
+        }
+    }
+
+    @media (min-width: @screen-sm) {
+        .oe_secondary_menu {
+            > li {
+                > a {
+                    margin-top: 2.5px;
+                    height: 25px;
+                }
+            }
+        }
+    }
+
+    .o_sub_menu > .o_sub_menu_content > .oe_secondary_menu > li.open > .dropdown-menu {
+
+        // Drop-down menu should be scrollable on small screens
+        overflow: auto;
+
+        // Headers at dropdown-menu shoud look different
+        > li.dropdown-header {
+            color: white;
+            text-decoration: none;
+            background-color: #8E8E8E;
```

Comment:
I think you'd better use the corresponding [Odoo](https://github.com/odoo/odoo/blob/10.0/addons/web/static/src/less/variables.less) or [Bootstrap](https://github.com/odoo/odoo/blob/10.0/addons/web/static/lib/bootstrap/less/variables.less) variables for color and background-color.